Prog/Preservation_database.sas

```diff
@@ -627,21 +627,299 @@ set test;
+label HOME_1_ConstructionType="HOME construction type";  
+label HOME_1_InacStatusDesc="Subsidy status description";  
+label HOME_2_ID="HOME subsidy ID";  
+label HOME_2_Status="Current status of HOME subsidy";  
+label HOME_2_ProgramName="Name of program";  
+label HOME_2_StartDate="Start date of subsidy";  
+label HOME_2_EndDate="Expiration date of subsidy";  
+label HOME_2_AssistedUnits="Total number of units covered by subsidy";  
+label HOME_2_ConstructionType="HOME construction type";  
+label HOME_2_InacStatusDesc="Subsidy status description";  
+label NumberActivePublicHousing="Total number of active Public Housing subsidies attached
+to property";
+label NumberInconclusivePublicHousing="Total number of inconclusive Public Housing subsidies
+attached to property";
+label NumberInactivePublicHousing="Total number of inactive Public Housing subsidies
+attached to property";
+label PH_1_ID="HUD assigned AMP number";  
+label PH_1_Status="Current status of Public Housing subsidy";  
+label PH_1_ProgramName="Name of program";  
+label PH_1_StartDate="Public housing subsidy start date";  
+label PH_1_EndDate="Expiration date of subsidy";  
+label PH_1_AssistedUnits="Total number of ACC units at property";  
+label PH_1_InacStatusDesc="Subsidy status description";  
+label PH_1_PhaCode="Subsidy code";  
+label PH_2_ID="HUD assigned AMP number";  
+label PH_2_Status="Current status of Public Housing subsidy";  
+label PH_2_ProgramName="Name of program";  
+label	PH_2_StartDate="Public housing subsidy start date";  
+label	PH_2_EndDate="Expiration date of subsidy";  
+label	PH_2_AssistedUnits="Total number of ACC units at property";  
+label	PH_2_InacStatusDesc="Subsidy status description";  
+label	PH_2_PhaCode="Subsidy code";  
+label	NumberActiveState="Total number of active state subsidies attached to
+property";
+label	NumberInconclusiveState="Total number of inconclusive state subsidies
+attached to property";
+label	NumberInactiveState="Total number of inactive state subsidies attached
+to property";
+label	State_1_ID="Unique state provided subsidy ID";  
+label	State_1_Status="Current status of State subsidy";  
+label	State_1_ProgramName="Name of program";  
+label	State_1_StartDate="Start date of subsidy";
+label	State_1_EndDate="Expiration date of subsidy";
+label	State_1_AssistedUnits="Total number of units covered by subsidy";
+label	State_1_InacStatusDesc="Subsidy status description";  
+label	State_1_ConstructionType="State construction type";  
+label	State_2_ID="Unique state provided subsidy ID";  
+label	State_2_Status="Current status of State subsidy";  
+label	State_2_ProgramName="Name of program";  
+label	State_2_StartDate="Start date of subsidy";
+label	State_2_EndDate="Expiration date of subsidy";
+label	State_2_AssistedUnits="Total number of units covered by subsidy";
+label	State_2_InacStatusDesc="Subsidy status description";  
+label	State_2_ConstructionType="State construction type";  
+label   COGregion="COG county";
```

Comment:
CBSACode
CensusTract
need labels. - please make sure to check the proc contents.